sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2962,12 +2962,12 @@ object functions {
   }
 
   /**
-   * Converts time string with the given pattern to timestamp.
+   * Converts time string with the given pattern to timestamp in the seconds precision.
```

Comment:
Just in case, are you ok with this change?

Reply:
Same with the others.